content/help/faq/distributed-tracing/mixer-role.md

```diff
@@ -0,0 +1,10 @@
+---
+title: What role does Mixer play in the Istio tracing story?
+weight: 90
+---
+
+By default, Mixer participates in tracing by generating its own spans for requests that are already selected for tracing by Envoy proxies. This enables operators to observe the participation of the mixer-based policy enforcement mechanisms within the mesh. If the `istio-policy` configuration is disabled mesh-wide, Mixer does not participate in tracing in this way.
+
+Mixer, operating as the `istio-telemetry` service, can also be used to generate trace spans for data plane traffic. Mixer’s Stackdriver adapter is the first adapter to support this capability. Soon, we will support Zipkin (and Zipkin API-compatible backends) in a similar fashion.
```

Comment:
Here and elsewhere you can simply by just saying "backends compatible with Zipkin trace format" as that catches the same in less words and doesn't limit you to POST (ex kafka is very reused and the more supportable way to forward trace traffic)

Reply:
I've removed that "Soon..." line and tried to reword the previous sentence. I'm aware of the SkyWalking remote adapter and of pending PRs around zipkin support. I suggest coming back and updating this after an initial commit with a link to a full list.  SG?